server/migrations/versions/5004a93f75d8_batchresulttallysheet.py

```diff
@@ -0,0 +1,85 @@
+# pylint: disable=invalid-name
+"""BatchResultTallySheet
+
+Revision ID: 5004a93f75d8
+Revises: cec7ecc73bd8
+Create Date: 2022-05-02 22:25:04.648947+00:00
+
+"""
+import uuid
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "5004a93f75d8"
+down_revision = "cec7ecc73bd8"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Echoing back my understanding, after this migration, we'll have `batch` entries that can reference multiple `batch_result_tally_sheet` entries that can further reference multiple `batch_result` entries -- is that correct?

Reply:
yep! you can see the declarative version that's easier to grok in models.py